Makefile

```diff
@@ -0,0 +1,35 @@
+ifneq (,$(wildcard ./.env))
+    include .env
+    export
```

Comment:
Are any of the variables used in a subshell? If not I think you can drop the export.